<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>车牌地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="plateAddrlineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>车牌端口：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="platePortlineEdit_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="platelabel_5">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="text">
         <string>车牌链接状态</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>箱号地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="containerAddrlineEdit_3"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>箱号端口：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="containerPortlineEdit_4"/>
      </item>
      <item>
       <widget class="QLabel" name="containerlabel_6">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="text">
         <string>箱号链接状态</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>透传数据：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Send485lineEdit_5"/>
      </item>
      <item>
       <widget class="QLineEdit" name="Send485EndlineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="send485pushButton_2">
        <property name="text">
         <string>显示数据</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="capturepushButton">
        <property name="text">
         <string>模拟抓拍</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="putpushButton">
        <property name="text">
         <string>手动抬杆</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Test</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>信息</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
